From: Randy Dunlap <rdunlap@infradead.org>
Date: Tue, 22 Oct 2019 20:57:06 -0700
Subject: reset: fix reset_control_ops kerneldoc comment
Git-commit: f430c7ed8bc22992ed528b518da465b060b9223f
Patch-mainline: 5.4-rc7
References: bnc#1151927 5.3.15

Add a missing short description to the reset_control_ops documentation.

Signed-off-by: Randy Dunlap <rdunlap@infradead.org>
[p.zabel@pengutronix.de: rebased and updated commit message]
Signed-off-by: Philipp Zabel <p.zabel@pengutronix.de>
Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 include/linux/reset-controller.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/linux/reset-controller.h
+++ b/include/linux/reset-controller.h
@@ -7,7 +7,7 @@
 struct reset_controller_dev;
 
 /**
- * struct reset_control_ops
+ * struct reset_control_ops - reset controller driver callbacks
  *
  * @reset: for self-deasserting resets, does all necessary
  *         things to reset the device
